src/family/family.resolver.spec.ts

```diff
@@ -136,4 +138,48 @@ describe('family Query', () => {
 
     await prisma.family.delete({ where: { id: family.id } });
   });
+
+  it('should create family and returns it', async () => {
+    const familyName = faker.name.fullName();
+
+    const query = gql`mutation { createFamily(input: { name: "${familyName}" }) { ... on DraftFamily { id status } } }`;
+
+    const result = await apolloServer.executeOperation({ query });
+
+    expect(result.errors).toBeFalsy();
+    expect(result.data).toBeTruthy();
+    expect(result.data?.['createFamily']?.status).toBe(FamilyStatus.DRAFT);
+    expect(result.data?.['createFamily']?.id).toBeTruthy();
+
+    await prisma.family.delete({
+      where: { id: result.data?.['createFamily']?.id },
+    });
+  });
+
+  it('should create two family with same name and different slug', async () => {
+    const familyName = 'some random big name which does not exist really';
+    const familySlug = slug(familyName);
+
+    const query = gql`mutation { createFamily(input: { name: "${familyName}" }) { ... on DraftFamily { slug } } }`;
+
+    const firstRequestResult = await apolloServer.executeOperation({ query });
+    const secondRequestResult = await apolloServer.executeOperation({
+      query,
+    });
+
+    const firstSlug = firstRequestResult.data?.['createFamily']?.slug;
+    const secondSlug = secondRequestResult.data?.['createFamily']?.slug;
+
+    expect(firstSlug).toBeTruthy();
+    expect(secondSlug).toBeTruthy();
+    expect(firstSlug).not.toEqual(secondSlug);
+
```

Comment:
I'm not sure we need to make sure family store in db and check it or not
but I do it in my ticket what is you opinion?
Is this  an overdo or not?